mlserver/handlers/dataplane.py

```diff
@@ -25,10 +27,15 @@ class DataPlane:
     servers.
     """
 
-    def __init__(self, settings: Settings, model_registry: MultiModelRegistry):
+    def __init__(
+        self,
+        settings: Settings,
+        model_registry: MultiModelRegistry,
+        response_cache: Optional[ResponseCache] = None,
+    ):
         self._settings = settings
         self._model_registry = model_registry
-
+        self._response_cache = response_cache
```

Comment:
Given that `response_cache` doesn't need to be shared outside of the `DataPlane` class, should we just create it here from `settings` instead of passing it as an arg?

Reply:
Thanks for the review. This comment makes a lot of sense to me. I will update the PR soon.

Reply:
Updated in [43dfc1c](https://github.com/SeldonIO/MLServer/pull/1440/commits/43dfc1c7587e738d377d437761fcb2bcade94854)